notebooks/stress-tests.ipynb

```diff
@@ -38,8 +38,8 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        run_threads(target_test = stress_test, url= url, execution_id = execution+1, n_threads=3, runs_per_threads=TEST_RUNS)
</pre><p>Can we not hardcode n_threads=3 but make it configurable through env var like TEST_RUNS?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Yes good point ! 
So I added a `TEST_N_THREADS` as a configurable env var.